hazelcast/src/main/java/com/hazelcast/wan/WanEventDistributedServiceCounters.java

```diff
@@ -30,7 +30,7 @@
  * This class may contain counters for a single WAN publisher or multiple
  * WAN publishers, depending on its usage.
  */
-public class DistributedServiceWanEventCounters {
```

Comment:
I think `WanEventCounters` name fit better for public api instead of `DistributedServiceWanEventCounters`. Since it implies a _not distributed_ counterpart but we don't have it. We can rename internal class `com.hazelcast.wan.impl.WanEventCounters` to something else like `WanEventCounterRegistry`.

Reply:
Changed both.